Applications/DataExplorer/Base/CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ add_library(QtBase
     ${HEADERS}
 )
 
-target_link_libraries(QtBase PUBLIC Qt4::QtGui)
+target_link_libraries(QtBase
+    Qt4::QtGui
+    logog
+    GeoLib)
```

Comment:
I think `logog` can be removed here, since `GeoLib` is linked with `MathLib` which is linked with `BaseLib` which is linked with `logog`.